### DISCUSSION TITLE
Is there any options for rtl?

### DISCUSSION THREAD

**Discussion by Calmway** [bcakmakoglu/vue-flow#773] [Q&A] [answered]
Hello everyone. Wanna ask about rtl (right to left)?
Maybe i did something wrong but all edges positioned incorrectly (separately from nodes)

**Comment by bcakmakoglu** [comment, marked as answer]
Have you checked the [Horizontal Flow](https://vueflow.dev/examples/horizontal.html) example? That should be what you're looking for, right?
